lib/src/analytics.dart

```diff
@@ -27,6 +29,18 @@ class Analytics {
       forceReset: forceReset ?? false,
     );
     initializer.run();
+
+    // Initialize the config handler class and check if the
+    // tool message and version have been updated from what
+    // is in the current file; if there is a new message version
+    // make the necessary updates
+    // ignore: unused_local_variable TODO: remove after increment method implemented
+    bool messagePrinted = false;
+    if (!_configHandler.parsedTools.containsKey(tool)) {
```

Comment:
Does this need a test?

Reply:
Good point, I was going to add them soon, but I will add them here first